packages/api/lib/reconciliationReport.js

```diff
@@ -2,7 +2,10 @@
 
 const { removeNilProperties } = require('@cumulus/common/util');
 const { constructCollectionId } = require('@cumulus/message/Collections');
+const Logger = require('@cumulus/logger');
+
 const { deconstructCollectionId } = require('./utils');
+const log = new Logger({ sender: '@api/lambdas/create-reconciliation-report' });
```

Comment:
Nit about above line is that we can also import `deconstructCollectionId` from `@cumulus/message/Collections`, but it's also kind of out of scope here

Reply:
Might as well fix it while we're thinking about it: 339615b0b